src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -107,13 +171,24 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({ lifecycleData }: Lifecy
           right: 50, // Adjusted to accommodate tooltip
           top: 50,
         }}
-        width={chartWidth}
+        // width={chartWidth}
+        // adjust this by number of items
+        height={900}
+        width={900}
       >
         <ChartAxis
           dependentAxis
           showGrid
-          tickFormat={(t: Date) => t.toLocaleDateString('en-US', { year: 'numeric' })}
+          tickFormat={(t: Date) => {
+            return t.toLocaleDateString('en-US', { year: 'numeric' });
+          }}
           tickValues={[
+            new Date('January 1 2017'),
```

Comment:
Don't need a lot of these dates, but we can adjust later. We want the tick values based on real data.

Reply:
For now i'll remove 2017 and 2018